Add link to the report that generated the notification

Cas-Ros opened the pull request.

Commit:
added link to the report than generated the notification

fniessink reviewed (changes requested):
Please use the `notification["report_name"]` in line 29 as anchor for the URL so no separate line is needed for the link to the report. 

Also, the `notification["report_name"]` isn't actually the report name, it's the report UUID. Maybe you can fix this as well?

Commit:
made changes accoding to feedback and modified logg message stating to
set an enviroment variable, this is nolonger needed as users can
configure this themselves

Commit:
made notification message multiline so the line is nolonger to long

fniessink commented on components/notifier/src/strategies/reds_per_report.py:
```diff
@@ -8,7 +8,7 @@ def reds_per_report(json) -> List:
     for report in json["reports"]:
         red_metrics.append({
-            "report_name": report["report_uuid"],
+            "report_uuid": report["report_uuid"],
```
```suggestion
            "report_uuid": report["report_uuid"],
            "report_title": report["title"],
```

fniessink commented on components/notifier/src/quality_time_notifier.py:
```diff
@@ -26,11 +26,11 @@ async def notify(log_level: int = None) -> None:
         for notification in notifications:
-            text = "\n\r" + notification["report_name"] + " contains " + str(notification["red_metrics"]) + \
-                   " red metrics" + "\n\r" + notification["url"]
+            text = "\n\r[" + notification["report_uuid"] + "](" + notification["url"] + \
```
```suggestion
            text = "\n\r[" + notification["report_title"] + "](" + notification["url"] + \
```

fniessink reviewed (changes requested):
You're using the report uuid as anchor for the URL, but I think the report title would be more useful. See suggested changes.

Commit:
Update components/notifier/src/strategies/reds_per_report.py

Co-authored-by: Frank Niessink <frank@niessink.com>

Commit:
Update components/notifier/src/quality_time_notifier.py

Co-authored-by: Frank Niessink <frank@niessink.com>

Commit:
updated unit tests with updated message parameters
